### ISSUE TITLE
Redirect does not append base path for non-absolute URLs

### ISSUE THREAD

**Issue by ivanhoe123** [mikecao/flight#12]
Hi Mike,

Redirect does not append base path for non-absolute URLs, so when you use Flight::redirect('somewhere'); and your installation is not in root, it won't redirect properly.

This is the fix - Flight.php, line 367:
(I don't know how to send differently, new to Github)

```
public static function _redirect($url, $code = 303) {
    // If this is not an absolute URL, add the base path
    if (!strpos($url, '//') || strpos($url, 'http://')) {
        $url = self::request()->base.'/'.$url;
    }
   (...)
```

Btw, thanks for such a cool framework :)

Cheers

**Comment by mikecao** [comment]
Thanks for reporting this. It is now fixed in the latest commit, fc10a50ca301b4c56c237ae4708bb6e885b51799.

**Comment by ivanhoe123** [comment]
I notice now that I double check the same thing; checking just the double slashes should be enough.

Thanks for adding this
